<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Looplorer 0.4.9</string>
  </property>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>911</width>
     <height>475</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButBkmrk">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>492</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>书签</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>492</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>新窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDevTools">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>492</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>开发者工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAbout">
   <property name="geometry">
    <rect>
     <x>746</x>
     <y>492</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
